src/service/plugins/sftp.js

```diff
@@ -317,6 +318,54 @@ var Plugin = GObject.registerClass({
         }
     }
 
+    /**
+     * Create a symbolic link referring to the device by name
+     */
+    _addSymlink(mount) {
```

Comment:
Since this function does IO, make it an `async` function and wrap the whole body in a single try-catch.